lib/abide_dev_utils/sce/generate/reference.rb

```diff
@@ -97,6 +97,25 @@ def generate(doc_title = 'Reference')
                 raise "Failed to generate markdown for control #{control.id}. Original message: #{e.message}"
               end
             end
+
+            if @module_name.split('-').last == 'sce_linux'
+              md.add_h1('For Controls Without Resource Data')
```

Comment:
This doesn't make sense in the context that customers are the ones reading it. They won't know what resource data is, or why controls would have it.